### ISSUE TITLE
Add ability to exclude specific keys from blocking

### ISSUE THREAD

**Issue by xerus2000** [rcaelers/workrave#194]
I like the ability to block input to stop myself from messing around with it. But I have encountered two problems with it:
1. I can't stop my music, which can be quite annoying if I am leaving my computer for the break and it is standing at an open space.
2. I had a drop-down terminal opened which went over the workrave break window, meaning I couldn't even postpone the break, I _had_ to wait.
Maybe it would make sense to unblock function keys and modifiers in general, so that you could also still for example close a window with Alt+F4 or log out with Ctrl+Shift+Backspace if you want to - these are not really inputs. I think the main things to be blocked should be letters, numbers and the mouse. Idk how easily that is to adjust, maybe that could also be an additional blocking option if you don't want to modify the existing one.
